### DISCUSSION TITLE
Alpha Coalition

### DISCUSSION THREAD

**Discussion by SultanRTG** [endless-sky/endless-sky#9313] [Q&A] [answered]
Are Alphas capable of change?  I saw a spaceport questline where the first Alpha went to the Coalition, and supposedly changed his ways after a conversation with a Drak/Archon. Which Archon was this referring to, and did the Alpha really change?

**Comment by Zitchas** [comment]
I would like to hope that they can. :)

**Comment by mOctave** [comment, marked as answer]
I'd say that anyone is capable of change. The Alpha obviously is changed to some extent in that mission, after all. I expect it would be far harder for an Alpha to adopt socially acceptable humanesque behaviour than it would be for a human, of course, but when it comes down to it, Alphas are sentient and so must be capable of some level of adaption.

I'm not sure there's any way to know which Archon it was. After all, how can we even know that the Archons are the same "ship" each time and not just given names based on their current station?
